opentasks/src/main/res/values/strings.xml

```diff
@@ -247,7 +249,10 @@
     <string name="accept">Accept</string>
     <string name="now">now</string>
 
-    <!-- Sharing intent -->
+    <!-- Incoming share -->
     <string name="share_as_task">Create Task</string>
 
+    <!-- Outgoing share -->
+    <string name="opentasks_share_footer">Shared by OpenTasks</string>
```

Comment:
I'm not certain about the preposition "by" in this case.
If I use OpenTasks to share a task with you, it has been "shared **by** me **with** you **xxx** OpenTasks". So what's the right preposition here?

Reply:
Maybe "from" ?  The other I thought about is "via", but I think "from" could be straightforward. "Shared from OpenTasks". What do you think?

Reply:
"via" would refer to the transport like in "via email" or "via SMS".
I've though about "from" as well, but it sounds strange to me. I'll try to ask a native speaker.

Reply:
According to https://ell.stackexchange.com/questions/141931/preposition-for-shared "via" is a valid option. So let's go with that.

Reply:
Updated to `Shared via OpenTasks`.